app/controllers/families_controller.rb

```diff
@@ -17,6 +17,17 @@ def create
     respond_with @family, location: -> { families_path }
   end
 
+  def edit
+    @family = Family.find(params[:id])
+  end
+
+  def update
+    @family = Family.find(params[:id])
```

Comment:
Consider pulling the finder query into a private method in the controller. @apalski can show you an example